tests/Microsoft.Identity.Test.Unit.net45/PublicApiTests/ConfidentialClientApplicationTests.cs

```diff
@@ -403,6 +417,114 @@ await app
             }
         }
 
+        [TestMethod]
+        [TestCategory("ConfidentialClientApplicationTests")]
```

Comment:
Do not add a category that is the same as the test class - it's just duplicate information.